clar2wasm/src/initialize.rs

```diff
@@ -11,8 +11,8 @@ use crate::linker::link_host_functions;
 use crate::wasm_utils::*;
 
 // The context used when making calls into the Wasm module.
-pub struct ClarityWasmContext<'a, 'b> {
-    pub global_context: &'a mut GlobalContext<'b>,
+pub struct ClarityWasmContext<'a, 'b, 'hooks> {
```

Comment:
Why does the `'hooks` lifetime need to come back?

Reply:
I'm not sure what you mean by "come back"
If it was here before, and was removed, I guess I could have failed a rebase 😬

Reply:
Thanks for catching it @obycode 
I updated [stacks-core](https://github.com/stacks-network/stacks-core/pull/4756/commits/abea0d349d43f46d3d73eb9e3bc0cb1663dc58f5) and this PR as well

I don't remember how I ended up adding this change

Reply:
Oh, maybe I'm misremembering, but I thought @krl had removed this lifetime a while back.

Reply:
Looking at my PRs (and considering how easy it was to remove it - again), I think you're right